setup-env/action.yaml

```diff
@@ -17,7 +17,7 @@ runs:
       shell: bash
     - name: Downcase repo
       run: |
-        echo "REPO_LOWER=${REPO,,}" >> ${GITHUB_ENV}
+        echo "REPO_LOWER=${REPO}" >> ${GITHUB_ENV}
```

Comment:
The 2 commas are the whole point of this method. They down case the repo variable.

Reply:
I was going to ask what they meant on Monday. It was my pattern recognition telling me this was odd.

Reply:
Now when I see it I know it's not a typo. What is it called so I can look it up, I am curious to know more about it.